hasweb/pages/site/tos.md

```diff
@@ -46,7 +50,7 @@ The terms of service specific to posting a listing are listed under the [Job Boa
 
 * Events
 
-HasGeek organizes events e.g. conferences, workshops, hacknights for the technology community, and incurs expenses for venue, food, snacks, stationery & printing, audio & video equipment as well as other rental or purchased equipment. HasGeek reserves entry to select events only to valid ticket holders. A ticket is valid only for the persons named on the ticket issued by us.
```

Comment:
I haven't removed the validity clause entirely. I have only removed the line "A ticket is valid only for the persons named on the ticket issued by us." which isn't true because we issue tickets in name of Company as well. 

Since the whole clause is a single line, Git thinks I've removed the entire line instead of a part of the line.